Gemfile

```diff
@@ -2,3 +2,9 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in rspec-graphql_matchers.gemspec
 gemspec
+
+if ENV['GRAPHQL_GEM_VERSION'] == '1.8'
+  gem 'graphql', '~> 1.8.0'
+elsif ENV['GRAPHQL_GEM_VERSION'] == '1.9'
```

Comment:
@severin why not just update `rspec-graphql_matchers.gemspec`:

```
spec.add_runtime_dependency 'graphql', '>= 1.8'
```
Instead of the current:
```
spec.add_runtime_dependency 'graphql', '>= 1.8', '< 1.9'
```
?